rush-plugins/rush-amazon-s3-build-cache-plugin/src/integration-tests/.gitignore

```diff
@@ -0,0 +1 @@
+data/.minio.sys
```

Comment:
What's this?

Reply:
These are some data folders used by min.io. They are created when booting up the docker-compose. I mount the minio bucket directory in the data folder so we can test fetching of the `data/rush-build-cache/testfile.txt` object. In the data folder minio also adds the `.minio.sys` metadata stuff.